Emby.Server.Implementations/Library/CoreResolutionIgnoreRule.cs

```diff
@@ -44,8 +48,15 @@ public bool ShouldIgnore(FileSystemMetadata fileInfo, BaseItem parent)
 
             if (IgnorePatterns.ShouldIgnore(fileInfo.FullName))
             {
-                return true;
-            }
+	        if(!_configurationManager.Configuration.IgnoreHiddenPaths)
```

Comment:
This seems a bit convoluted. I think line 49 should just be changed to
```
             if (IgnorePatterns.ShouldIgnore(fileInfo.FullName) && _configurationManager.Configuration.IgnoreHiddenPaths)
```

Reply:
Thanks again @cvium. I don't know how to decide what files to (un)ignore from `IgnorePatterns.cs`. 
I feel that unix .dots should not be on that list. This is an attempt to keep the default behaviour as is but allow an override in case it's useful.